src/schema/models.js

```diff
@@ -1,8 +1,20 @@
 import mongoose from 'mongoose'
-import User from './userSchema'
+import * as Models from './index'
```

Comment:
you can only do ` import Models from './index' `

Reply:
this import should not be used here anymore